dart/lib/src/transport/http_transport.dart

```diff
@@ -61,7 +60,7 @@ class HttpTransport implements Transport {
     } else {
       _options.logger(
         SentryLevel.debug,
-        'Event ${event.eventId} was sent suceffully.',
+        'Event ${event.eventId} was sent successfully.',
```

Comment:
merge main into this branch, this is fixed already

Reply:
✅